docs/orchestration/index.md

```diff
@@ -342,4 +201,4 @@ stack {
 }
 ```
 
-An error will be reported.
+In this situation, a conflict arises causing the execution to enter failure mode. An error message will be reported in such an instance.
```

Comment:
Not specifically about your change here, as the old text wasn't ideal also, but it'd be better to explain that it's a fatal error, not just an error message.